dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -146,14 +146,23 @@ export default defineComponent({
      */
     addKpisOfOneDatasetToTableModel(lksgData: LksgData, dataId: string) {
       for (const [categoryKey, categoryObject] of Object.entries(lksgData) as [string, object | null]) {
-        if (categoryObject == null) continue;
+        if (categoryObject == null) {
+          lksgData[categoryKey as string] = {};
```

Comment:
Why change category and subcategory from `null` to an empty object?

Reply:
if category or subcategory is null then we change it to empty object yes